src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -370,19 +431,13 @@ export const ImportAccounts = ({ chainId, onClose, isOpen }: ImportAccountsProps
     setIsSubmitting(false)
   }, [toggledAccountIds, accounts, dispatch, walletDeviceId])
 
-  const handleDoneClick = useCallback(async () => {
-    await handleUpdateAccounts()
+  const handleCommit = useCallback(() => {
```

Comment:
Not sure about the naming here, but this is not really a "Done" (will be fired on close too) and is really just a wrapper to not await the async logic. Happy to use something that reads better!